client/packages/programs/src/JsonForms/common/components/Text.tsx

```diff
@@ -41,7 +45,12 @@ type Options = z.infer<typeof Options>;
 // Validates the option and parses the pattern into RegEx
 const useOptions = (
   options?: Record<string, unknown>
-): { errors?: string; options?: Options; pattern?: RegExp } => {
+): {
+  errors?: string;
+  options?: Options;
+  pattern?: RegExp;
+  useDebounce?: boolean;
```

Comment:
I don't think `useDebounce` is a separate prop, is it? It should be on the `options` object.